gen-ai/inference/stable-diffusion-rayserve-gpu/download_models.ipynb

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Let's remove this notebook. Update the [website](https://awslabs.github.io/data-on-eks/docs/gen-ai/inference/GPUs/stablediffusion-gpus) instructions to show saving the model from inside a python k8s job that runs the `download_models.py` below to save the model to s3 using the mountpoint.